helm/wts/values.yaml

```diff
@@ -71,8 +71,10 @@ metricsEnabled: false
 externalSecrets:
   # -- (string) Will override the name of the aws secrets manager secret. Default is "Values.global.environment-.Chart.Name-creds"
   dbcreds:
-  # -- (string) Will override the name of the aws secrets manager secret. Default is "wts-g3auto".
-  wtsG3auto:
```

Comment:
We do not want to remove this value

Reply:
Bad copy paste from when I pulled the change in from master and didn't want to deal with merge conflicts. Will re-add.